src/django/api/reports.py

```diff
@@ -22,11 +29,80 @@ def monthly_promoted_name_and_address():
                     month = h[1].strftime('%Y-%m')
                     data[month] = data.get(month, 0) + 1
     return [['month', 'facilities promoted'],
-            sorted(data.items(), key=lambda x: x[0])]
+            sort_by_first_column(data.items())]
+
+
+def geocoding_time_without_queue(date_format):
+    temp_data = dict()
+    listitems = FacilityListItem.objects.values_list(
+        'created_at', 'processing_results')
+    for f in listitems:
```

Comment:
To avoid using subscripts, I think that we can use tuple destructuring here to get some more readable variable names.

```python
for (created_at, processing_results) in listitems:
```